src/controllers/deploy.go

```diff
@@ -32,11 +37,13 @@ func deploy(callbackID string, data map[string]interface{}) {
 		log.Errorf("%s - %v", actionLog, err)
 		history.StoreAction(actionLog)
 		slack.PostChatMessage(channel, fmt.Sprintf("%s\nError: %s\n", actionLog, err.Error()), nil)
+		discord.PostActionMessage(actionLog, callbackID)
```

Comment:
don't duplicate it here and on line 46 below

Put this after the if-else block.